oneflow/api/python/framework/tensor_functions.cpp

```diff
@@ -747,15 +967,15 @@ static PyObject* PyTensorObject_to_global(PyObject* self, PyObject* args, PyObje
   HANDLE_ERRORS
   const auto& tensor = PyTensor_Unpack(self);
   PyObject* result = NULL;
-  if (tensor->is_global())
-    result = PyTensorObject_global_to_global(self, args, kwargs);
-  else {
-    result = PyTensorObject_local_to_global(self, args, kwargs);
-  }
-  if (PyErr_Occurred()) { throw py::error_already_set(); }
-  return result;
+  if (tensor->is_global()) result = PyTensorObject_global_to_global(self, args, kwargs);
+}
```

Comment:
```suggestion
  if (tensor->is_global()){ result = PyTensorObject_global_to_global(self, args, kwargs);
} else ...
```

这里是笔误？原代码里的 `}` 好像是和函数的左括号匹配了